js/muzima.js

```diff
@@ -163,6 +163,9 @@ $(document).ready(function () {
             validForm = validForm && $.fn.customValidationCheck();
         }
         if (validForm) {
+	    $(this).find('input:text').each(function(){
```

Comment:
@nribeka, we want to prompt warning immediately if inputting future date for encounter date across all form as standard.  I agree to put changes in muzima.js so it applies to all.